content/en/docs/concepts/workloads/autoscaling.md

```diff
@@ -106,6 +106,14 @@ While the Cluster Proportional Autoscaler scales the number of replicas of a wor
 adjusts the resource requests for a workload (for example a Deployment or DaemonSet) based on the number of nodes and/or cores
 in the cluster.
 
+### HTTP request based autoscaling
+
+It is also possible to scale workloads based on concurrent HTTP requests.
+[Knative](https://knative.dev/) uses this approach to dynamically scale workloads
+including dynamically including scale to zero.
+
+Knative is a CNCF incubating project.
```

Comment:
Thanks, but we avoid linking to projects outside of Kubernetes. We do that only for things clusters pretty much need to have.

See https://k8s.io/docs/contribute/style/content-guide/#third-party-content

/hold

pending guidance (from SIG Docs leadership) that we can make an exception.

It's tricky because the page already mentions KEDA, which is also not part of Kubernetes.